pkg/scheduler/framework/plugins/examples/multipoint/multipoint.go

```diff
@@ -14,49 +14,51 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package examples
+package multipoint
 
 import (
+	"encoding/json"
 	"fmt"
 
 	"k8s.io/api/core/v1"
-	plugins "k8s.io/kubernetes/pkg/scheduler/plugins/v1alpha1"
+	framework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
 )
 
 // MultipointCommunicatingPlugin is an example of a plugin that implements two
 // extension points. It communicates through pluginContext with another function.
 type MultipointCommunicatingPlugin struct{}
 
-var _ = plugins.ReservePlugin(MultipointCommunicatingPlugin{})
+var _ = framework.ReservePlugin(MultipointCommunicatingPlugin{})
+var Name = "multipoint-communicating-plugin"
```

Comment:
Will this `Name` be used elsewhere? It seems a bit inappropriate. And it seems that there is no inspection by golint.

Reply:
The Name is supposed to be used as the "key" for registering plugins in "Registry".